packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -1,6 +1,10 @@
 const fs = require('fs');
 const path = require('path');
-const { getCwd } = require('@hubspot/local-dev-lib/path');
+const {
+  getCwd,
+  sanitizeFileName,
```

Comment:
I decided to rename this function when moving it to local-dev-lib, because it makes it more readable and reusable in the future. We're basically treating the project name as a directory/file name.